include/core.hpp

```diff
@@ -33,7 +33,9 @@ namespace color_coded
 
     inline std::string temp_dir()
     {
-      static auto const dir(fs::temp_directory_path() / "color_coded/");
+      static auto const dir(fs::temp_directory_path() /
+                            (std::string("color_coded-") +
+                             std::string(getenv("USER"))));
```

Comment:
Optionally, while you're at it, you could pull the "color_coded-<user>" out into a separate `user_dir` and then simplify the definition of `dir`.

Remember that the original path had a trailing slash; the new one currently does not.